server/model/status_page.js

```diff
@@ -46,7 +47,14 @@ class StatusPage extends BeanModel {
      */
     static async renderHTML(indexHTML, statusPage) {
         const $ = cheerio.load(indexHTML);
-        const description155 = statusPage.description?.substring(0, 155) ?? "";
+
+        let description155 = "";
+        if (statusPage.description) {
+            description155 = marked(statusPage.description)
+                .replace(/<[^>]+\>g, "")
```

Comment:
Fixed removing the wrong backslash in 5e8477044429ff091f15bf2e5ae2ff48612ddeca

```suggestion
                .replace(/<[^>]+>/g, "")
```